src/control/lib/control/pool.go

```diff
@@ -1200,6 +1204,7 @@ func ListPools(ctx context.Context, rpcClient UnaryInvoker, req *ListPoolsReq) (
 		p.PoolLayoutVer = resp.PoolLayoutVer
 		p.UpgradeLayoutVer = resp.UpgradeLayoutVer
 		p.setUsage(resp)
+		p.RebuildStat = fmt.Sprintf("%s", resp.Rebuild.State)
```

Comment:
minor - A simplification suggestion:

```suggestion
		p.RebuildStat = resp.Rebuild.State.String()
```